projects/bunicorn/index.js

```diff
@@ -27,6 +28,31 @@ query GET_TOTAL_STABLE_TVL($block: Int) {
 }
 `;
 
+
+async function getTotalFarmTVL(timestamp, ethBlock, chainBlocks) {
+  try {
+    const block = chainBlocks.bsc
+    const result = await sdk.api.abi.multiCall({
+      calls: [
+        {
+          target: BUNI_CONTRACT_ADDRESS,
+          params: [MASTERCHET_CONTRACT_ADDRESS]
+        }
+      ],
+      block,
+      abi: abi.balanceOf,
+      chain: 'bsc'
+    })
+    const balance = result.output[0] && result.output[0].output;
+    const buniPrice = await getBuniPrice(chainBlocks);
```

Comment:
Does this need to be amended? afaik getBuniPrice is not defined

Reply:
I fixed it